scripts/unix/dependencies-bower.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+###
+# Copyright 2016 resin.io
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+set -u
+set -e
+
+function check_dep() {
+  if ! command -v $1 2>/dev/null 1>&2; then
+    echo "Dependency missing: $1" 1>&2
+    exit 1
+  fi
+}
+
+check_dep bower
+
+function usage() {
+  echo "Usage: $0"
+  echo ""
+  echo "Options"
+  echo ""
+  echo "    -x <install prefix>"
+  echo "    -p production install"
+  exit 0
+}
+
+ARGV_PREFIX=""
+ARGV_PRODUCTION=false
+
+while getopts ":x:fp" option; do
+  case $option in
+    x) ARGV_PREFIX=$OPTARG ;;
+    p) ARGV_PRODUCTION=true ;;
+    *) usage ;;
+  esac
+done
+
+INSTALL_OPTS="--allow-root"
+
+if [ "$ARGV_PRODUCTION" == "true" ]; then
+  INSTALL_OPTS="$INSTALL_OPTS --production"
+fi
+
+if [ -n "$ARGV_PREFIX" ]; then
+  pushd "$ARGV_PREFIX"
+  bower install $INSTALL_OPTS
+  popd
+fi
+
+bower install $INSTALL_OPTS
```

Comment:
Is this right: `bower install` being run twice?? Or is the second one supposed to be inside an `else` clause?